### ISSUE TITLE
Do you plan on adding support for GridView?

### ISSUE THREAD

**Issue by EnderShadow** [edvin/tornadofx-controlsfx#8]

**Comment by edvin** [comment]
I'm not actively adding support to the controlsfx-project, but by looking at how the integration is done you should be able to add support for GridView and supply a PR :)

**Comment by thomasnield** [comment]
Yes. This project is largely community contributed. If something is missing we will happily look at PRs.
